jina/drivers/index.py

```diff
@@ -54,9 +53,18 @@ def _apply_all(self, docs: 'DocumentSet', *args, **kwargs) -> None:
 class SparseVectorIndexDriver(VectorIndexDriver):
     """An alias to have coherent naming with the required SparseVectorSearchDriver """
 
-    @staticmethod
-    def _get_documents_embeddings(docs: 'DocumentSet'):
-        return docs.all_sparse_embeddings
+    @property
+    def exec_sparse_cls_type(self) -> str:
+        return self.exec.sparse_cls_type
+
+    def _get_documents_embeddings(self, docs: 'DocumentSet'):
```

Comment:
oh I see

Reply:
But should the driver be storing this spare class type? Maybe it should still be static and just pass the type as another argument? I don't see it as being part of the Driver's expected knowledge

Reply:
oh yes, that would make sense too. We could keep it static like this

Reply:
default argument being dense representation class?

Reply:
It would not be used in the Dense case. It is handy to access dense embbedding via a property